services/environment.js

```diff
@@ -1,10 +1,11 @@
-var cfenv = require('cfenv');
-var appEnv = cfenv.getAppEnv();
-var env = process.env.APP_ENV || 'development';
-var serviceCredsKey = `federalist-${env.toLowerCase()}-env`;
-var federalistCreds = appEnv.getServiceCreds(serviceCredsKey) || {};
+const cfenv = require('cfenv');
```

Comment:
just linting fixes in this file. not really related to any logic changes in this PR, just happened to take a peak at it.